packages/shared/lib/contexts/index.ts

```diff
@@ -0,0 +1,2 @@
+export * as onboarding from './onboarding'
+export * as settings from './settings'
```

Comment:
I think this makes sense since this module directory is different than `core` and `auxiliary` in that it's plural, meaning it houses modules of the same type. If we didn't specify them on this level then the auto import statements would default to `... from '@contexts'`.

But looking at it now, the `index.ts` file in `lib/core` only exports the `router` so perhaps we can ignore the barrel files on that level and just force more explicit import paths.